portal/patient/fwk/libs/verysimple/String/SimpleTemplate.php

```diff
@@ -56,8 +56,8 @@ static function TextToHtml($txt)
         $txt = str_replace('&amp;', '&', $txt);
         
         // Ajdusts links - anything starting with HTTP opens in a new window
-        // $txt = str_ireplace("<a href=\"http://","<a target=\"_blank\" href=\"http://",$txt);
-        // $txt = str_ireplace("<a href=http://","<a target=\"_blank\" href=http://",$txt);
+        // $txt = str_ireplace("<a href=\"http://","<a rel="noopener" target=\"_blank\" href=\"http://",$txt);
+        // $txt = str_ireplace("<a href=http://","<a rel="noopener" target=\"_blank\" href=http://",$txt);
```

Comment:
For the follow up commit recommend:
```
        // $txt = str_ireplace("<a href=\"http://","<a rel=\"noopener\" target=\"_blank\" href=\"http://",$txt);
        // $txt = str_ireplace("<a href=http://","<a rel=\"noopener\" target=\"_blank\" href=http://",$txt);
```
(note this is commented out, but might as well fix it)

Reply:
I think I got it!